webapp/src/js/services/enketo.js

```diff
@@ -329,9 +346,19 @@ angular.module('inboxServices').service('Enketo',
       }
     };
 
+    var replaceMarkup = function(doc) {
+      $('.question-label, .question > .or-hint', doc.html).each(function () {
+        $(this).html(Markdown.basic($(this).html()));
+      });
+      return doc;
+    };
+
     var renderForm = function(selector, form, instanceData, editedListener, valuechangeListener) {
       return Language().then(language => {
         return transformXml(form)
+          .then(doc => {
+            return replaceMarkup(doc);
+          })
```

Comment:
Don't forget to use the auto return style where possible, eg: `.then(doc => replaceMarkup(doc))`

Reply:
Done.